app/workers/playlist_import_worker.rb

```diff
@@ -1,9 +1,13 @@
 class PlaylistImportWorker
+  include Sidekiq::Worker
+  sidekiq_options queue: :spotify_import
+
   def perform(user_id)
     user = SpotifyUser.find(user_id)
 
     user.playlists.fetch(:items).each do |api_playlist|
```

Comment:
using `user.playlists` should access the playlists stored on the database. If we want to access the playlists in the api , the method name should be clear about that.

Also, Consider to add a concern which handles the connection to the spotify api, and from the model just set whatever constants necesary to make it work